Examples/Python/tests/test_navigation.py

```
@@ -40,3 +40,9 @@ def test_navigation_policy_factory_add_multiple():
             .add(acts.TryAllNavigationPolicy)
             .add(acts.TryAllNavigationPolicy)
         )
+
+
+def test_try_all_arguments():
```

Ruff

Missing return type annotation for public function `test_try_all_arguments`

Add return type annotation: `None`

(ANN201)